activerecord/lib/active_record/transactions.rb

```diff
@@ -409,6 +409,7 @@ def clear_transaction_record_state
       # Force to clear the transaction record state.
       def force_clear_transaction_record_state
         @_start_transaction_state.clear
+        @mutations_before_first_save = nil
```

Comment:
I assume clearing the first save mutations after a successful commit is acceptable, and not within the scope of `ActiveModel` to care about. There's probably a better place to put this. Where is it?